L1Trigger/TrackFindingTracklet/src/Tracklet.cc

```diff
@@ -405,7 +406,9 @@ std::string Tracklet::fullmatchdiskstr(int disk) {
   } else {
     tcid.set(TCIndex_, 7, true, __LINE__, __FILE__);
   }
+  const FPGAWord& stubr = diskresid_[disk - 1].stubptr()->r();
   std::string oss = tcid.str() + "|" + tmp.str() + "|" + diskresid_[disk - 1].fpgastubid().str() + "|" +
+                    ((stubr.nbits() == 4) ? ("00000000" + stubr.str()) : stubr.str()) + "|" +
```

Comment:
Please avoid magic number in "stubr.nbits() == 4).

Reply:
Done.